stdlib/Random/src/Xoshiro.jl

```diff
@@ -21,6 +21,13 @@ multiple interleaved xoshiro instances).
 The virtual PRNGs are discarded once the bulk request has been serviced (and should cause
 no heap allocations).
 
+The `seed` may be an integer or a vector of `UInt32` integers.
```

Comment:
I actually don't want to commit yet to supporting `Vector{UInt32}` as seeds. This support is perhaps useful because 
MersenneTwister (so the old GLOBAL_RNG) accepted these seeds. So we might want to silently support that for `TaskLocalRNG()` only. I've some more ideas on how to iterate on seeding stuff, so don't want to commit just yet.

```suggestion
The `seed` may be an integer. 
```

Reply:
Ah perfect! So my idea was that low entropy seeds like integers should ultimately go through SHA. But we could consider that when you pass an array of 4 UInt64 (or 8 UInt32) words, these 256 bits already have sufficient entropy and can be used directly to initialize the state. So my idea was to allow any array of integer bits types, but only of the correct size (256 bits in total), and to skip SHA in this case. Tuples of integer bit integers could also be accepted. What do you think?

Reply:
I have a local branch actually implementing `seed!("the seed")`! but didn't push it yet because there are enough PR going.